### ISSUE TITLE
iPadOS 13.3.1 run error

### ISSUE THREAD

**Issue by AnthoDingo** [ColdGrub1384/Pisth#27]
Hello,

Can't run on iPad OS 13.3.1. Build is ok, but run is ko.

First error : 
```
dyld: Library not loaded: @rpath/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0
  Referenced from: /private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Pisth
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0: code signature invalid for '/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0'

	/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0: code signature invalid for '/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0'

	/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0: stat() failed with errno=1
	/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0: code signature invalid for '/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0'

	/private/var/containers/Bundle/Application/2716E7F4-88D7-43BD-BDA7-AF49954EBF1C/Pisth.app/Frameworks/ActionSheetPicker_3_0.framework/ActionSheetPicker_3_0: stat() failed with errno=1
```

Several projects encounter the same problem : https://github.com/Alamofire/Alamofire/issues/3051
I have fixed this bug by editing Pods\Podfile and replacing
`use_frameworks!`

by 
`use_modular_headers!`

After cleaning projet, update pod and installing lib, I got this error : 
`dyld: Library not loaded: @rpath/Pisth_Shared.framework/Pisth_Shared
  Referenced from: /private/var/containers/Bundle/Application/4A07DBED-8386-46DA-B925-F5350452AC01/Pisth.app/Pisth
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/4A07DBED-8386-46DA-B925-F5350452AC01/Pisth.app/Frameworks/Pisth_Shared.framework/Pisth_Shared: code signature invalid for '/private/var/containers/Bundle/Application/4A07DBED-8386-46DA-B925-F5350452AC01/Pisth.app/Frameworks/Pisth_Shared.framework/Pisth_Shared'`

Sorry, I'm newest on iOS/iPadOS.

Do you have solution ?

**Comment by manuduarte** [comment]
Same issue, only runs in simulator

**Comment by AnthoDingo** [comment]
Fixed with iOS 13.4
